test/app.testx.js

```diff
@@ -2,7 +2,6 @@ const dotenv = require('dotenv');
 dotenv.config('../.env');
```

Comment:
hmm, why rename to `.testx.js`? What does it mean?

Reply:
sorry was skipping app tests when building 2fa.test.js, will fix in upcoming commits